discovery-package/src/main/resources/bin/start.sh

```diff
@@ -67,6 +67,13 @@ if [ `uname` = "OS/390" ]; then
     QUICK_START=-Xquickstart
 fi
 
+DISCOVERY_LOADER_PATH=${COMMON_LIB}
+
+if [[ ! -z ${ZWE_DISCOVERY_SHARED_LIBS} ]]
+then
+    DISCOVERY_LOADER_PATH=${ZWE_GATEWAY_SHARED_LIBS},${DISCOVERY_LOADER_PATH}
```

Comment:
`ZWE_DISCOVERY_SHARED_LIBS` maybe?
```suggestion
    DISCOVERY_LOADER_PATH=${ZWE_DISCOVERY_SHARED_LIBS},${DISCOVERY_LOADER_PATH}
```